airbyte-integrations/connectors/source-postgres/src/test-integration/java/io/airbyte/integrations/io/airbyte/integration_tests/sources/XminPostgresSourceWithOldPostgresAcceptanceTest.java

```diff
@@ -0,0 +1,13 @@
+package io.airbyte.integrations.io.airbyte.integration_tests.sources;
+
+/**
+ * This is testing xmin sync with an older version of PostgreSQL which reclassifies ctid syncs back to xmin
+ * Initial load of table is done with xmin.
+ */
+public class XminPostgresSourceWithOldPostgresAcceptanceTest extends XminPostgresSourceAcceptanceTest{
```

Comment:
This test seems to be the same? Is there a way to have a test where : 
1. CTID Categorized streams emit CTID state
2. For recategorized streams, we assert that CTID state is not emitted (thereby confirming the initial sync here took a different path)

Reply:
alright - added unit test to make validate xmin state in case we reclassified.